lib/kubernetes-deploy/ejson_secret_provisioner.rb

```diff
@@ -137,21 +137,16 @@ def decrypt_ejson(key_dir)
     end
 
     def fetch_private_key_from_secret
-      secret = run_kubectl_json("get", "secret", EJSON_KEYS_SECRET)
+      out, err, st = @kubectl.run("get", "secret", EJSON_KEYS_SECRET, output: "json")
+      raise EjsonSecretError, err unless st.success?
+      result = JSON.parse(out)
+      secret = result.fetch('items', result)
```

Comment:
This is redundant now. The original `run_kubectl_json` didn't know if it would be getting one or multiple secrets, so had to account for the latter case. Since we only ever get one secret at a time where it used to be called, we can remove it